scripts/functions

```diff
@@ -1305,6 +1305,24 @@ CT_DoBuildTargetTuple() {
     CT_ARCH_TARGET_LDFLAGS="${CT_ARCH_TARGET_LDFLAGS} ${CT_ARCH_ENDIAN_LDFLAG}"
 }
 
+# This function determines the target tuple for a given set of compiler
+# flags, using either GCC's multiarch feature (if supported) or using
+# fallback architecture-specific functions.
```

Comment:
... except that some multilib target configurations in GCC do not support multiarch, in which case --enable-multiarch is a no-op. mingw target is one such example. If that's the case, GCC will still process --print-multiarch option, but will just report an empty string. Yes, it's convoluted and GCC's behavior makes me cringe, too.